src/MoBi.R/MinimalImplementations/HistoryManagerFactory.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Autofac;
+using OSPSuite.Core.Commands.Core;
+using OSPSuite.Infrastructure.Serialization.ORM.History;
+
+namespace MoBi.R.MinimalImplementations
+{
+   public class HistoryManagerFactory: IHistoryManagerFactory
+   {
+      private readonly IComponentContext _context;
+      public HistoryManagerFactory(IComponentContext context)
```

Comment:
why don't you get the contaienr as parmeter? What's an IComponentContext? We never use Autofac stuff directly. Feels weird to me